front/src/applications/operationalStudies/components/Scenario/ScenarioDescription.tsx

```diff
@@ -33,7 +62,22 @@ const ScenarioDescription = ({
 
       <div className="scenario-description">
         {scenario.description && (
-          <div className="scenario-details-description">{scenario.description}</div>
+          <div
+            ref={descriptionRef}
+            title={isOpenedDescription ? '' : scenario.description}
```

Comment:
Hm, is this intentional? This displays a huge tooltip on hover.

Reply:
done